### ISSUE TITLE
Did this get adopted?

### ISSUE THREAD

**Issue by stuartpb** [substack/mmmify#5]
I just stumbled on this repo when @ForbesLindesay starred it. It sounds fairly sensible (though I like #3's Lua-style export suggestion and #1's async semantics stuff), and everything I've heard about ES6 modules... well, http://blog.izs.me/post/25906678790/on-es-6-modules#module-essay just about sums it up (specifically the "Problems With The Current Spec" section).

However, that post says that the proposal has changed since it was written, and that the changes address most of the problems described in that post. What changes are these? Everything I can find about ES6 modules, even the stuff that appears to have been written in the last two years, looks like it's geared toward a style of modules that has *all* the problems described in that post. What am I missing here?

**Comment by ForbesLindesay** [comment]
No, this didn't get adopted, but the proposal for ES6 modules was improved dramatically, and this repo probably has a part to play in that.

**Comment by stuartpb** [comment]
What changed? Like I said, I don't see any differences between the examples people were begrudging here and the ones they're using in the latest documentation.
On Dec 24, 2015 3:45 PM, Forbes Lindesay <notifications@github.com> wrote:No, this didn't get adopted, but the proposal for ES6 modules was improved dramatically, and this repo probably has a part to play in that.
